apps/webapp/Dockerfile.slim

```
@@ -0,0 +1,61 @@
+WORKDIR /app
+
+# Copy pruned files
+COPY --from=base /app/out/json/ .
+COPY --from=base /app/out/pnpm-lock.yaml ./pnpm-lock.yaml
+COPY --from=base /app/out/full/ .
+
+# Install dependencies
+RUN pnpm install --shamefully-hoist
+
+# Build shared package first
+RUN cd packages/shared && pnpm run build
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)